mgmt/RecordsConfig.cc

```diff
@@ -1212,6 +1212,8 @@ static const RecordElement RecordsConfig[] =
   //##############################################################################
   {RECT_CONFIG, "proxy.config.ssl.enabled", RECD_INT, "0", RECU_RESTART_TS, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.ssl.session.ticket.enabled", RECD_INT, "1", RECU_RESTART_TS, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
```

Comment:
``RECU_RESTART_TS`` means you need to restart Traffic Server when you change this setting. AFAICT you just need to touch ``ssl_mulicert.config`` and reload config, so this probably should be ``RECU_DYNAMIC``.